src/styleguide/_patterns/02-molecules/03-form-group/01-form-group.njk

```diff
@@ -0,0 +1,13 @@
+{%- from 'atoms-label' import label -%}
```

Comment:
I suggest making the folder name more generic. Perhaps `03-forms/...` rather than `03-form-group/...`. If/when we have other form related molecules we can then pop them in there too. :-)